### PR TITLE
ESLint changes to eslint-test-vat2

### PR DESCRIPTION
**Pull request by katelynsills**
## Errors
screen shot 2019-02-21 at 5 33 56 pm

## Fixes
1. Add global Vow, Flow
2. Remove unused imports
2. Change all to CamelCase
3. Change unnamed functions to arrow style
4. allow intentional unused vars and args

### CODE REVIEW

File: test/test-vat2.js [hunk 1]
```diff
@@ -751,49 +836,49 @@ test('third-party Vow gets resolved', async t => {
...
-  const req = s.makeRequire({'@agoric/nat': Nat, '@agoric/harden': true});
+  /* eslint-disable-next-line no-unused-vars */
+  const req = s.makeRequire({ '@agoric/nat': Nat, '@agoric/harden': true });
 
   const ALICE = 'ALICE';
-  const alice_src = funcToSource(t5_alice);
+  const aliceSrc = funcToSource(t5Alice);
   const vatALICE = await buildVat(
     s,
-    req,
```
Comment (warner) [inline test/test-vat2.js]: I think this 'req' parameter needs to stay.

File: test/test-vat2.js [hunk 2]
```diff
@@ -751,49 +836,49 @@ test('third-party Vow gets resolved', async t => {
...
+  const aliceArgv = { bob: vatALICE.createPresence('vatBOB/0') };
+  /* eslint-disable-next-line no-unused-vars */
+  const aliceRoot = await vatALICE.initializeCode('vatALICE/0', aliceArgv);
 
   const BOB = 'BOB';
-  const bob_src = funcToSource(t5_bob);
+  const bobSrc = funcToSource(t5Bob);
   const vatBOB = await buildVat(
     s,
-    req,
```
Comment (warner) [inline test/test-vat2.js]: and here

File: test/test-vat2.js [hunk 3]
```diff
@@ -751,49 +836,49 @@ test('third-party Vow gets resolved', async t => {
...
-  const bob_root = await vatBOB.initializeCode('vatBOB/0', bob_argv);
+  const bobArgv = { carol: vatBOB.createPresence('vatCAROL/0') };
+  const bobRoot = await vatBOB.initializeCode('vatBOB/0', bobArgv);
 
   const CAROL = 'CAROL';
-  const carol_src = funcToSource(t5_carol);
+  const carolSrc = funcToSource(t5Carol);
   const vatCAROL = await buildVat(
     s,
-    req,
```
Comment (warner) [inline test/test-vat2.js]: and here

Comment (katelynsills) [reply test/test-vat2.js]: good catch!

**Pull request merged**